vic/vic_run/src/runoff.c

```diff
@@ -148,17 +150,11 @@ runoff(cell_data_struct  *cell,
            Initialize Variables
         **************************************************/
         for (lindex = 0; lindex < options.Nlayer; lindex++) {
-            Ksat[lindex] = soil_con->Ksat[lindex] /
-                           global_param.runoff_steps_per_day;
-
```

Comment:
why is this section taken out?

Reply:
It wasn't eliminated - it was moved up to lines 88-89.  The reason for it is that Ksat, max_moist, and resid_moist are all temporary variables that can be set at the very beginning.